provision.sh

```diff
@@ -16,7 +15,7 @@ echo "---------------------------------------------"
 
 # install heroku toolbelt
 echo "-------------- Installing heroku toolbelt -------------------------"
-#wget -O- https://toolbelt.heroku.com/install-ubuntu.sh | sh
+wget -O- https://toolbelt.heroku.com/install-ubuntu.sh | sh
```

Comment:
Actually it was a connection error that happened and the key wasn't properly downloaded :stuck_out_tongue: 
The script works - that is part of the Python vagrant.